lib/Elastica/Query/Fuzzy.php

```diff
@@ -35,10 +35,10 @@ public function __construct($fieldName = null, $value = null)
      */
     public function setField($fieldName, $value)
     {
-        if (!is_string($value) or !is_string($fieldName)) {
+        if (!is_string($value) || !is_string($fieldName)) {
             throw new InvalidException('The field and value arguments must be of type string.');
         }
-        if (count($this->getParams()) > 0 and array_shift(array_keys($this->getParams())) != $fieldName) {
+        if ($this->getParams() && key($this->getParams()) != $fieldName) {
```

Comment:
@merk, I stay logic `as is` before - only refactoring, and test it before in `sandbox`